bazel/execution/parse.go

```diff
@@ -152,6 +153,48 @@ func ExtractOpFromJson(opBytes []byte) (*longrunning.Operation, error) {
 	return op, nil
 }
 
+// Converts a longrunning.Operation to a Json-encoded []byte. Because an Operation is not natively compatible
+// with json.Marshal, this does the minimum necessary extraction of nested data and wrapping with custom
+// types so that it can be used with Marshal.
+func OperationToJson(op *longrunning.Operation) ([]byte, error) {
+	if op == nil {
+		return nil, nil
+	}
+
+	eom, st, res, err := ParseExecuteOperation(op)
+	if err != nil {
+		return nil, err
+	}
+
+	jsonResult := &operationResultWrapper{}
+	if res != nil {
+		jsonResult.Error = nil
```

Comment:
why not jsonResult.Error = st (are error stats vs result mutually exclusive?)

Reply:
yep - done